tests/integration/components/setup-global-a11y-hooks-test-test.ts

```diff
@@ -0,0 +1,52 @@
+import { module, test } from 'qunit';
```

Comment:
Can we rename this file? I think `setup-global-a11y-hooks-for-render-test.ts` seems like a reasonable name.